src/types/type-declarations.ts

```diff
@@ -62,6 +62,10 @@ export interface Output {
 /* Google Analytics */
 export interface GoogleAnalyticsParams {
   id: string;
+  /**
+   * The name of the dataLayer object. Defaults to 'dataLayer'.
+   */
+  l?: string;
```

Comment:
dataLayername... using "l", since that's the param expected by us

Reply:
Why `l`? Is that the name we're using to map to Nuxt or something that the gtag docs suggest?

Reply:
this is preferable for me as well, would like to see the Nuxt Scripts options reflect this

Reply:
Yes nuxt script options needs to use l otherwise, we'll have l and dataLayername in the options. Also this is better for consistency